content/support-training.html

```diff
@@ -0,0 +1,173 @@
+---
+title: Support & Training
+layout: jumbotron
+---
+
+<div class="container-fluid training">
+  
+  <div class="row gray-bg">
+    <div class="container">
+    <div class="col-md-12 doc-content">
+      <h1>Support & Training</h1>
+      <p>
+        Prometheus is an open source project. Training resources on this page are 
+        listed for your convenience only. These resources are independently 
+        created, and their content is not governed or verified by the
+        Prometheus project's maintainers.
+      </p>
+      
+      <p>If you'd like to add your training resource, see <a href="#add">Add a resource</a>.
+    </div>
+    </div>
+  </div>
+
+
+  <div class="row">
+    <div class="container">
+      <div class="col-md-12 doc-content training-team">
+        <h2>Training</h2>
+        <p>
+          These training resources are created and delivered independently 
+          by members of the Prometheus project team. Note that these do operate for a fee. This list is provided in alphabetical order.
+        </p>
+      </div>
+    </div>
+    <div class="container equal">
+      <div class="col-sm-12 col-md-6 doc-content panel-col">
+          <div class="panel panel-default">
+            <div class="panel-body">
+              <a href="https://training.robustperception.io/" target="_blank">
+                <h3>Learn how to Monitor with Prometheus</h3>
+              </a>
+              <p>A set of courses. Master master Prometheus, including the data model, PromQL, configuration, Grafana, alerting, instrumenting and practical deployment.</p>
+              <p>Created by Brian Brazil and delivered via <a href="https://training.robustperception.io/" target="_blank">Robust Perception Training</a>.
+            </div>
+          </div>
+      </div>
+      <div class="col-sm-12 col-md-6 doc-content panel-col">
+          <div class="panel panel-default">
+            <div class="panel-body">
+              <a href="https://training.linuxfoundation.org/training/monitoring-systems-and-services-with-prometheus-lfs241/" target="_blank">
+                <h3>Monitoring Systems and Services with Prometheus</h3>
+              </a>
+              <p>This course leads new Prometheus users through many of its major features, best practices, and use cases. Course participants are expected to have basic experience with 
+                Linux/Unix system administration, as well as some development experience in Go and/or Python.</p>
+                <p>Created by Julius Volz and delivered via <a href="https://training.linuxfoundation.org/" target="_blank">Linux Foundation Training</a></p>
+
+            </div>
+          </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row gray-bg">
+    <div class="container">
+      <div class="col-md-12 doc-content training-community">
+        <h2>Commercial support</h2>
+        <p>
+          These resources are created by members of the Prometheus community who are 
```

Comment:
I'd add some description of what these resources are, like we have on the current Community page. At the same time, some of the people here *are* affiliated with the project. So how about just taking the existing Community page blurb? ("This is a list of third-party companies and individuals who provide products or services related to Prometheus. Prometheus is an independent open source project which does not endorse any company. The list is provided in alphabetical order.").